redhat-appstudio/service-provider-integration-operator
# Add CSP header to HTML resources

### What does this PR do?
Adds an CSP header for static HTML-s. It allows use inline styles, images from redhat domain, and denying everything else, including framing


### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/1651062/222129856-2505168c-5214-4bfb-b2a7-b12181ebaed0.png)


### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-374

### How to test this PR?

 Run through OAuth, make sure all page requests have a CSP header included in response.

```
quay.io/redhat-appstudio/pull-request-builds:spi-operator-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```
